src/IssueCreator/IssueManager.cs

```diff
@@ -22,8 +22,31 @@ public class IssueManager
         private GitHubClient _githubClient;
         private FileLogger _fileLogger;
         private readonly MemoryCache _cache = new MemoryCache(new MemoryCacheOptions());
-        private readonly string _cacheFolder;
-
+        private readonly string _cacheFolder;
+        public delegate void IssueLoadedDelegate(object sender, IssueObject issue);
+        public event IssueLoadedDelegate IssueLoadedForTemmplateEvent;

```

Comment:
We could also call this something like `IssueTemplateLoaded` and the other would be `BrowseableIssueLoaded` and `IssueLoaded` ? 

I'm thinking that the main use case for the issue being loaded is when an issue is just loaded for non-specific reasones (like the association or the template loading).

Thoughts?

Reply:
I like your naming much better, and it's more idiomatic for event names.